### PR TITLE
imv: add page

### PR DESCRIPTION
**Pull request by ikks**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/imv.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+- View in fullscreen mode:
+
+`imv -f {{path/to/image.ext}}`
+
+- View images recursively in a directory:
+
+`imv --slideshow {{path/to/image.ext}}`
+
+- Open multiple images via stdin:
```
Comment (spageektti) [inline pages/linux/imv.md:19]:
```suggestion
- Open multiple images via `stdin`:
```

File: pages/linux/imv.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+`find . -type f -name "*.svg" | imv`
```
Comment (spageektti) [inline pages/linux/imv.md:21]:
```suggestion
`find . -type f -name "{{*.svg}}" | imv`
```

File: pages/linux/imv.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+- Make a slideshow from a directory showing each image for 10 seconds:
+
+`imv -t 10 {{path/to/directory}}`
+
+- View multiple images from the web:
+
+`curl -Osw '%{filename_effective}\n' 'http://www.example.com/[1-10].jpg' | imv`
```
Comment (spageektti) [inline pages/linux/imv.md:29]:
```suggestion
`curl -Osw '%{filename_effective}\n' '{{http://www.example.com/[1-10].jpg}}' | imv`
```

**Review by gutjuri — COMMENTED** [review]
Thanks, I have some suggestions

File: pages/linux/imv.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+`imv {{path/to/image1.ext path/to/image2.ext ...}}`
+
+- View in fullscreen mode:
+
+`imv -f {{path/to/image.ext}}`
+
+- View images recursively in a directory:
+
+`imv --slideshow {{path/to/image.ext}}`
```
[the remark this answers is not in the document]
Comment (ikks) [reply pages/linux/imv.md]: Totally agree, please let me know if what has just been committed makes more sense. Thank you :)

**Review by gutjuri — APPROVED** [review]
Thanks for your changes!

**Review by spageektti — APPROVED** [review]

**Pull request merged**